code/src/main/java/frc/robot/commands/VisionAdjustCommand.java

```diff
@@ -0,0 +1,57 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018-2019 FIRST. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.commands;
+
+import frc.robot.common.NullLogger;
+import com.google.inject.Inject;
+import frc.robot.common.*;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.Subsystem;
+import edu.wpi.first.networktables.*;
+import edu.wpi.first.networktables.NetworkTableEntry;
+import edu.wpi.first.networktables.NetworkTableInstance;
+
+public class VisionAdjustCommand extends TraceableCommand {
+    private NetworkTable m_table;
+    private int m_theta;
+
+    public VisionAdjustCommand(final ILogger logger) {
+        super(logger);
+        m_table = NetworkTableInstance.getDefault().getTable("limelight");
+        m_table.getEntry("pipeline").setNumber(0);
+    }
+
+    @Override
+    public void initialize() {
+
+    }
+
+    public void execute() {
+
+        // network table values
+        NetworkTableEntry tx = m_table.getEntry("tx");
+        NetworkTableEntry ty = m_table.getEntry("ty");
+        NetworkTableEntry ta = m_table.getEntry("ta");
```

Comment:
the calls to NetworkTables would be better behind a class ie LimelightSensor or something. This makes it easier to mock this out. It also means the data can be used in more than one place.

Actually making it a subsystem will make it easier to keep polling the values.